static/templates/recipient_row.hbs

```diff
@@ -50,11 +50,8 @@
                     {{/if}}
                 {{/if}}
 
-                {{#if topic_is_resolved}}
-                    <i class="fa fa-check on_hover_topic_unresolve recipient_bar_icon" data-topic-name="{{topic}}" data-tippy-content="{{t 'Mark as unresolved' }}" role="button" tabindex="0" aria-label="{{t 'Mark as unresolved' }}"></i>
-                {{else}}
-                    <i class="fa fa-check on_hover_topic_resolve recipient_bar_icon" data-topic-name="{{topic}}" data-tippy-content="{{t 'Mark as resolved' }}" role="button" tabindex="0" aria-label="{{t 'Mark as resolved' }}"></i>
-                {{/if}}
+                {{! resolve topic icon }}
+                <i class="fa fa-check on_hover_topic_resolve recipient_bar_icon" data-topic-name="{{topic}}" data-tippy-content="{{#if topic_is_resolved}}{{t 'Mark as unresolved' }}{{else}}{{t 'Mark as resolved' }}{{/if}}" role="button" tabindex="0" aria-label="{{#if topic_is_resolved}}{{t 'Mark as unresolved' }}{{else}}{{t 'Mark as resolved' }}{{/if}}"></i>
```

Comment:
Removing `on_hover_topic_unresolve` makes the click handler written for this widget no longer effective.